### PR TITLE
(1000) Add person names to bookings

### PR DESCRIPTION
**Pull request by pezholio**
This adds a person's name when viewing Bookings on the premises page. Before we didn't have access to this information via the API, but now we do, I think it makes sense to show this information upfront. I've also taken the opportunity to add some quality of life improvements to better match the patterns we're using in services and views.

## Before

before

## After

after

### CODE REVIEW

**Comment by pezholio** [comment]
[Add person names to bookings](https://trello.com/c/THnPcw1E/1000-add-person-names-to-bookings)

File: server/utils/bookingUtils.test.ts [hunk 1]
```diff
@@ -92,8 +92,6 @@ describe('bookingUtils', () => {
     })
   })
 
-  describe('currentResidentsToTableRows', () => {})
-
```
Comment (bryangaledxw) [inline server/utils/bookingUtils.test.ts]: Think this is the mirror of the stray added line in previous commit

File: server/form-pages/apply/schema.json [hunk 2]
```diff
@@ -940,19 +940,35 @@
           "type": "object",
           "properties": {
             "additionalNeeds": {
-              "type": "array",
```
Comment (bryangaledxw) [inline server/form-pages/apply/schema.json:943]: Is this a tidy-up that could be in its own commit?

File: cypress_shared/pages/manage/premisesShow.ts [hunk 3]
```diff
@@ -7,6 +7,7 @@ import { DateFormats } from '../../../server/utils/dateUtils'
 export default class PremisesShowPage extends Page {
   constructor(private readonly premises: ApprovedPremises) {
     super(premises.name)
+    cy.screenshot()
```
Comment (bryangaledxw) [inline cypress_shared/pages/manage/premisesShow.ts:10]: Is this leftover debugging?

**Review by bryangaledxw — APPROVED** [review]
Looks good, just a few minor comments

**Review by bryangaledxw — APPROVED** [review]
